app/controllers/api/v1/items/search_controller.rb

```diff
@@ -3,11 +3,26 @@ def show
     attribute = params.keys.first
     keyword = params[attribute]
     if attribute == "name" || attribute == "description"
-      item = Item.where("items.#{attribute} ILIKE ?", "%#{keyword}%").limit(1)
+      item = Item.find_by("items.#{attribute} ILIKE ?", "%#{keyword}%")
```

Comment:
find_by returns the object, not an array of objects allowing for the JSON payload returned to be processed by the rails driver (FE)